--- freeradius-2.1.9+dfsg/raddb/eap.conf~	2010-05-24 07:40:58.000000000 +0200
+++ freeradius-2.1.9+dfsg/raddb/eap.conf	2010-05-30 13:41:36.000000000 +0200
@@ -156,7 +156,7 @@
 			cadir = ${confdir}/certs
 
 			private_key_password = whatever
-			private_key_file = ${certdir}/server.pem
+			private_key_file = ${certdir}/server.key
 
 			#  If Private key & Certificate are located in
 			#  the same file, then private_key_file &
